src/content/docs/infrastructure/host-integrations/installation/container-auto-discovery-host-integrations.mdx

```diff
@@ -359,6 +359,6 @@ For example, a container that exposes the 80 and 443 private ports via the publi
 
 ### Mapping ports by protocol name
 
-In addition to the indexing from the previous section, ports can be also referred by its protocol name (e.g. `tcp`, `udp`, ...). For example, `${discovery.ports.tcp}` , `${discovery.ports.udp}`, etc...
+In addition to the indexing from the previous section, ports can be also referred by its protocol name (such as `tcp`, `udp`, etc). For example, `${discovery.ports.tcp}`, `${discovery.ports.udp}`.
```

Comment:
@austin-schaefer, I think you need a period after the "etc" before the closing parenthesis.